packages/console/src/components/Checkbox/CheckboxGroup/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { type AdminConsoleKey } from '@logto/phrases';
+import classNames from 'classnames';
+
+import DynamicT from '@/components/DynamicT';
+
+import Checkbox from '../Checkbox';
+
+import * as styles from './index.module.scss';
+
+type Option<T extends string> = {
```

Comment:
Just `type Option<T>`

Reply:
It's intended to limit the `T` extends from `string` since we don't want to support complex types yet.

Reply:
You don't have to limit it again in here, since you already have that at line 22: `function CheckboxGroup<T extends string>`.
The generic type `T` is then carried from component type definition to `Props<T>`, and finally to `Option<T>`.

Reply:
Oh, I get it, updated.